foundry.toml

```diff
@@ -6,13 +6,13 @@
   bytecode_hash = "none"
   evm_version = "paris"           # See https://www.evmdiff.com/features?name=PUSH0&kind=opcode
   fuzz = { runs = 1_000 }
-  via-ir = true
```

Comment:
might prefer profiles over globally turning off to allow for via-ir when deploying

Reply:
although I see you have it in the hardhat to toggle on/off